File: serial_lib/src/main/java/com/kongqw/serialportlibrary/listener/OnOpenSerialPortListener.java
package com.kongqw.serialportlibrary.listener;

import com.kongqw.serialportlibrary.enumerate.SerialPortEnum;
import com.kongqw.serialportlibrary.enumerate.SerialStatus;

import java.io.File;

/**
 * 打开串口监听
 */
public interface OnOpenSerialPortListener {

    void openState(SerialPortEnum serialPortEnum, File device, SerialStatus status);

}

File: serial_lib/src/main/java/com/cl/serialportlibrary/listener/OnOpenSerialPortListener.kt
package com.cl.serialportlibrary.listener;

import com.cl.serialportlibrary.enumerate.SerialPortEnum;
import com.cl.serialportlibrary.enumerate.SerialStatus;

import java.io.File;

/**
 * 打开串口监听
 */
public interface OnOpenSerialPortListener {

    void openState(SerialPortEnum serialPortEnum, File device, SerialStatus status);

}
